project/src/components/offers/offers.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+import { Offer } from '../../types/offers';
+import CitiesCard from '../card/card';
+
+
+type OffersProps = {
+  offers: Offer[];
+}
+
+function Offers({ offers }: OffersProps): JSX.Element {
+  const [activeCard, setActiveCard] = useState<number | null>(null);
```

Comment:
Если 1 параметр не нужен можно опустить временно
```suggestion
  const [, setActiveCard] = useState<number | null>(null);
```